core/server/data/migrations/versions/4.0/23-add-primary-key-brute-migrations-lock.js

```diff
@@ -0,0 +1,11 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const {addPrimaryKey} = require('../../../schema/commands');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    logging.info('Adding a primary key for the brute and migrations_lock tables');
+
+    await addPrimaryKey('brute', 'key', knex);
+
+    await addPrimaryKey('migrations_lock', 'lock_key', knex);
```

Comment:
The `migrations_lock` table is not "owned" by Ghost. It is created and managed by the knex-migrator. Imo, this migration should be happening inside of the knex-migrator's [migrations](https://github.com/TryGhost/knex-migrator/tree/fdd58cb911b39103029ad8e977384ee0029ca97a/migrations).

Reply:
good point 💯

Reply:
it has been removed ✅